docs/automated-tests/test-execution/quick-run-baseurl.md

```diff
@@ -1,29 +1,32 @@
 ---
-title: BaseURL Quick Access
-page_title: BaseURL Quick Access
-description: "Test Studio Quick test Execution. Test Studio test run. Quick access the BaseURL. Quick Set BaseURL on project level. "
+title: BaseUrl Quick Access
+page_title: BaseUrl Quick Access
+description: "Test Studio Quick test Execution. Test Studio test run. Quick access the BaseUrl. Quick Set BaseUrl on project level. "
 position: 3
 ---
-# Quick Set BaseURL for a Project
+# Quick Access to BaseUrl
```

Comment:
Don't change the capitalization. Preserve the capitalization of the original UI element. See https://testdocs.telerik.com/style-guide/element-formatting#guidelines